uvls/src/core/check.rs

```diff
@@ -253,6 +253,35 @@ pub fn classify_error(root: Node, source: &Rope) -> ErrorInfo {
             };
         }
     }
+    if Regex::new(r"^\d+")
```

Comment:
Now the quickfix is available, but it results in a fix that treats the attribute declaration as part of the feature name.  The regex (or whatever recognizes the name) should only match up until the firrst space or closing-quote appears.

<img width="275" alt="2023-10-10 15_49_51- Extension Development Host  ● imtest uvl - uvl - Visual Studio Code" src="https://github.com/Universal-Variability-Language/uvl-lsp/assets/131375895/1fc6348b-92ee-4b14-bb29-97b0ee57966f">   

quickfixes result in: 
`Feature {attrtest 1}_3` or `"3Feature {attrtest 1}"`

Reply:
Fixed it. Pls try again